Xamarin.PropertyEditing.Mac/Controls/Custom/ChannelEditor.cs

```diff
@@ -0,0 +1,298 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CoreAnimation;
+using CoreGraphics;
+using Xamarin.PropertyEditing.Drawing;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public abstract class ChannelEditor
```

Comment:
Does this need to be public?

Reply:
nope, there are a couple others I'll also fix